meilisearch/_httprequests.py

```diff
@@ -27,7 +27,7 @@ def send_request(
         self,
         http_method: Callable,
         path: str,
-        body: Optional[Union[Dict[str, Any], List[Dict[str, Any]], List[str], str]] = None,
+        body: Optional[Union[Dict[str, Any], Sequence[Mapping[str, Any]], List[str], str]] = None,
```

Comment:
Lets make both of these `Dict`s a `Mapping`

```suggestion
        body: Optional[Union[Mapping[str, Any], Sequence[Mapping[str, Any]], List[str], str]] = None,
```